spec/unit/prepare.spec.js

```diff
@@ -81,9 +82,9 @@ function mockGetIconItem (data) {
     }, data);
 }
 
-describe('prepare', () => {
-    describe('updateIcons method', function () {
-    // Rewire
+describe('prepare', function () {
+    describe('updateIcons', function () {
+        // Rewire
```

Comment:
Arrow functions shouldn't converted to `function ()` style.

Additionally this change appears to be unrelated to the problem this PR is trying to solve. PRs should be concise to only include changes required to address a problem.

Reply:
all non related changes have been reverted